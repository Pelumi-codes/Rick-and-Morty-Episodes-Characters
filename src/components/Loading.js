import React from "react";
import { Box, Text } from "@chakra-ui/react";

const Loading = () => {
  return (
    <Box my="20%">
      <Text textAlign="center" fontWeight="800" fontSize="16px">
        Loading....
      </Text>
    </Box>
  );
};

export default Loading;
