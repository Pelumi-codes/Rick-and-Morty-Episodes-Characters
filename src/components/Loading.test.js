import React from "react";
import renderer from "react-test-renderer";

import  Loading from "./Loading";
it("renders list of characters", () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});