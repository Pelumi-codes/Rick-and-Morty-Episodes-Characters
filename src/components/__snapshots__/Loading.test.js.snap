// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders list of characters 1`] = `
<div
  className="css-ap8j7p"
>
  <p
    className="chakra-text css-1knpu6j"
  >
    Loading....
  </p>
</div>
`;
